### PR TITLE
Survery Form for code review trials

### PR DESCRIPTION
**Pull request by Titus-Kiplagat**
- GitHub workflow added
- Linter errors fixed
- index.html added for the survey form
- style.css added for the survey form
- README.md file added

### CODE REVIEW

**Comment by Titus-Kiplagat** [comment]
# STATUS:  CHANGES REQUIRED  ♻️

Hello @Titus-Kiplagat, 

I acknowledge the great effort you've put into building the design for this milestone. The overall work is commendable; however, there are a few detailed issues that require your attention before proceeding to the next project. ♻️

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great UI design. 💻 
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Additionally, I would like to offer some optional suggestions. While not critical to the approval of this PR, I highly recommend considering them as they have the potential to enhance the quality of your code._

Cheers, and Happy coding!👏👏👏

If you have any questions or encounter any uncertainties, please feel free to ask in the PR thread. Tagging me will ensure that I receive a notification and can provide prompt assistance.

**_Lastly, I kindly ask that you refrain from opening a new Pull Request for re-reviews. Instead, please use the same Pull Request that was initially submitted, regardless of its status, unless explicitly instructed otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Titus-Kiplagat** [comment]
Please, consider doing changes in the following areas:
- [ ] Please, change the background color of the class "container". In your design you used rgba(39, 39, 66, 0.6) and the freecodecamp template they have used rgba(27, 27, 50, 0.8).
- [ ] Please, make changes in your input and textarea fields to have the right font family, font size, and line height as per the freecodecamp design.
- [ ] Please, also make changes to your header section. They have a different font weight.

File: README.md [hunk 1]
```diff
@@ -1 +1,8 @@
 # Survey-Form-Review
+> This is Microverse Trials Project used to apply code review skills.
+
+# Acknowledgements
+I would like to give credit to list of contributors:
+
+- Family.
+- Microverse Team.
```
Comment (Titus-Kiplagat) [inline README.md:8]: I would like to commend your efforts in creating the README for the project. It's great to see that you have identified the collaborators involved, which is an important aspect. However, I would like to suggest an enhancement that could further improve the overall quality of the project.

- You can use this [README template](https://github.com/microverseinc/readme-template). It is well descriptive but kindly customize it to your project.

 - [ ] I kindly request you to consider incorporating this template into the README, taking the opportunity to provide a clear overview of the project, highlight its key features, and guide potential users on installation and usage. Additionally, including instructions on how to contribute to the project will encourage community involvement and collaboration.

Comment (bobb-Rob) [reply README.md:8]: I must say that your review comments are professional, very polite, and contain good reasons for your suggestions, well done.

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @Titus-Kiplagat, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**